src/protocol/chromium-bidi.ts

```diff
@@ -87,7 +88,13 @@ export namespace Network {
   }
 }
 
-export type Command = (WebDriverBidi.Command | Cdp.Command) & {
+export type Command = (
+  | WebDriverBidi.Command
+  | Cdp.Command
+  | ({
+      id: WebDriverBidi.JsUint;
```

Comment:
Is `id` added because the permissions is an extension, and it relies on the BiDi spec to have the `id`, so it is not generated in types?

Reply:
Yes, the ID is part of the main spec and it is not defined explicitly in the permissions spec CDDL.

Reply:
Added